dependent/bundesgesundheitsblatt-gesundheitsforschung-gesundheitsschutz.csl

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="de-DE">
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer-fachzeitschriften-medizin -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
     <title>Bundesgesundheitsblatt - Gesundheitsforschung - Gesundheitsschutz (German)</title>
-    <title-short>Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz</title-short>
     <id>http://www.zotero.org/styles/bundesgesundheitsblatt-gesundheitsforschung-gesundheitsschutz</id>
     <link href="http://www.zotero.org/styles/bundesgesundheitsblatt-gesundheitsforschung-gesundheitsschutz" rel="self"/>
-    <link href="http://www.zotero.org/styles/springer-fachzeitschriften-medizin-psychologie" rel="independent-parent"/>
-    <link href="http://www.springer.com/medicine/journal/103?detailsPage=pltci_1426209" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
```

Comment:
Yes, that is fine, see https://github.com/citation-style-language/styles/issues/1583#issuecomment-110794540